# Fix for Gentoo bug #625316 (spamassassin bug #7367)

--- a/spamc/configure.pl	2017/02/13 02:14:42	1782716
+++ b/spamc/configure.pl	2017/02/13 02:16:44	1782717
@@ -66,7 +66,7 @@
   # Do the same thing as for the preprocessor below.
   package version_h;
   my $Z = $0;
-  local $0    = "version.h.pl";
+  local $0    = "./version.h.pl";
   local @ARGV = ();
   # Got to check for defined because the script returns shell error level!
   unless (defined do $0) {
